src/current/_data/releases.yml

```diff
@@ -6807,3 +6807,30 @@
     docker_arm_limited_access: false
   source: true
   previous_release: v23.2.11
+
+- release_name: v24.3.0-alpha.1
```

Comment:
I think adding this entry will cause Kathryn some manual work when she generates the actual release notes. Does leaving it off break anything?

Reply:
TBH I’m not sure, because this process required a lot of trial and error and further testing means more builds, and more time to leave this matter open. I’m inclined to merge and then test that question tomorrow off main and remove what I can.